### DISCUSSION TITLE
Canvas Lock for scrolling is a good idea

### DISCUSSION THREAD

**Discussion by Wladefant** [personalizedrefrigerator/js-draw#46] [Ideas & Feature Requests]
I would suggest adding either a canvas lock like samsung notes 
![SmartSelect_20231026_151244_Samsung Notes](https://github.com/personalizedrefrigerator/js-draw/assets/79611093/f79f2337-f5dd-4c69-be27-741dfe487c78)
or another option would be to remove the finger paint support and only paint with a pen. Or put it in settings where you can choose between pen support OR finger and Pen support.

Right now even while I am drawing I often draw with my skin

**Comment by personalizedrefrigerator** [comment]
There's currently "touchscreen panning" for this, but a canvas lock would also be useful.
<img src="https://github.com/personalizedrefrigerator/js-draw/assets/46334387/e9915d4f-8f28-4c93-af80-fd80391a8ae5" alt="screenshot: Hand tool menu open, touchscreen panning enabled"/>

**Reply by Wladefant** [reply]
Nice. I am not exactly able to understand what touchscreen panning is

**Reply by personalizedrefrigerator** [reply]
It should probably be labeled "disable touch drawing", "draw only with pen", or something similar.

**Reply by Wladefant** [reply]
Ah nice, it seems like you though about everything. I am not a native english speaker, but I havent heard about touchscreen panning, so yes renaming would be nice. I would suggest naming it **pen-only mode** OR **pen-only input**
